sunkit_image/utils/decorators.py

```diff
@@ -54,3 +57,20 @@ def inner(*args, **kwargs) -> np.ndarray | GenericMap:
         return inner
 
     return decorate
+
+
+def register_coalignment_method(name):
```

Comment:
I might have suggested this be moved here but I think it should go with the coalignment code. It's specific to that part of the codebase only.

Reply:
Alright